Tasks/KubernetesManifestV0/src/actions/delete.ts

```diff
@@ -0,0 +1,39 @@
+"use strict"; 
+
+import tl = require('vsts-task-lib/task'); 
+import { Kubectl } from "utility-common/kubectl-object-model"; 
+import * as utils from "../utils/utilities";
+import * as TaskInputParameters from '../models/TaskInputParameters';
+
+const TASK_INPUT_DELETE_BY_FILENAME = "filenames";
+const TASK_INPUT_DELETE_BY_RESOURCES_AND_NAMES = "resourcesAndNames";
+const TASK_INPUT_DELETE_BY_RESOURCES_AND_LABEL_SELECTORS = "resourcesAndLabelSelector";
```

Comment:
Is this really necessary for single time use?

Reply:
Purpose is to keep constants at 1 place together.